chevah/brink
# [#2659] Fix python elevated run.

Problem
=======

paver python test runner is stupid and will call elevated test even when they are explicitly skipped.

Why we got this
============

no tests

Changes
=======

Fix code to look for elevated only at start of module

How to test
========

reviewers: @alibotean 

check that changes make sense


Should not call elevated:
```
paver test_python normal.test_something --exclude=elevated
```

Should call elevated:

paver test_python elevated.normal.test_something

thanks!
